integration/spark/src/main/java/org/apache/spark/sql/CarbonVectorProxy.java

```diff
@@ -0,0 +1,554 @@
+/*
```

Comment:
I can see other files are moved, but why this file shows as added instead of moved ?

Reply:
CarbonVectorProxy.java have an indent issue, it led to many changes (about 400 lines).
I will use "git mv" to try again.

Reply:
@ajantha-bhat 
I try the following two ways in one commit, but it gets the same result.
1. git mv <old file> <new file> and modify new file 
2. modify old file and git mv <old file> <new file>

Finally, I raise two commits, you can review one by one in this PR....>_<